src/containers/auth/register/Register.js

```diff
@@ -0,0 +1,100 @@
+import React from "react";
```

Comment:
Lets move all hardcoded values to the helper file

Reply:
Fixed